### ISSUE TITLE
Operation package will be name replaced with version[0-9]+ if the tag contains v[0-9]+

### ISSUE THREAD

**Issue by rxnew** [go-swagger/go-swagger#3143]
## Problem statement

Since [v0.31.0](https://github.com/go-swagger/go-swagger/releases/tag/v0.31.0), if the operation tag contains `v[0-9]+`, the operation package name will be replaced with `version[0-9]+`. For example, even if the tag is `aaav1bbb`, it will be replaced with `version1`. It is not possible to avoid the problem by using `x-go-operation-tag`. This bug was introduced by https://github.com/go-swagger/go-swagger/pull/3043.

I have confirmed that this problem can be avoided with https://github.com/rxnew/go-swagger/commit/67580be6a4e9c532a26e4ea798e92d8546390b77.

## Swagger specification

2.0

## Steps to reproduce

1. Specify `aaav1bbb` in the operation tags
2. Run `swagger generate server`

## Environment
swagger version: 0.31.0  
go version: 1.23.1

**Comment by fredbi** [comment]
Yes I believe we should fix this and perform the replacement only when the "vnnn" is a suffix.
